sentry-android-gradle-plugin/src/main/groovy/io/sentry/android/gradle/SentryPlugin.groovy

```diff
@@ -329,18 +334,44 @@ class SentryPlugin implements Plugin<Project> {
                         if (buildTypeProperties.has(SENTRY_ORG_PARAMETER)) {
                             args.add("--org")
                             args.add(buildTypeProperties.get(SENTRY_ORG_PARAMETER).toString())
+
+                            nativeArgs.add("-o")
+                            nativeArgs.add(buildTypeProperties.get(SENTRY_ORG_PARAMETER).toString())
                         }
                         if (buildTypeProperties.has(SENTRY_PROJECT_PARAMETER)) {
                             args.add("--project")
                             args.add(buildTypeProperties.get(SENTRY_PROJECT_PARAMETER).toString())
+
+                            nativeArgs.add("-p")
+                            nativeArgs.add(buildTypeProperties.get(SENTRY_PROJECT_PARAMETER).toString())
+                        }
+
+                        nativeArgs.add("${project.projectDir}/build")
+
+                        if (extension.includeNativeSource) {
+                            nativeArgs.add("--include-sources")
                         }
 
+                        project.logger.info("cli args: ${args.toString()}")
+
+                        project.logger.info("nativeArgs args: ${nativeArgs.toString()}")
+
                         if (Os.isFamily(Os.FAMILY_WINDOWS)) {
                             commandLine("cmd", "/c", *args)
                         } else {
                             commandLine(*args)
                         }
 
+                        project.logger.info("args executed.")
+
+                        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                            commandLine("cmd", "/c", *nativeArgs)
+                        } else {
+                            commandLine(*nativeArgs)
+                        }
```

Comment:
btw I'm not sure if here is the right place to execute this command, as it may happen that NDK build is not done yet, probably hooking in the NDK build task is the way to go, but a bit error-prone on every new release.